examples/games/breakout.rs

```diff
@@ -323,15 +323,14 @@ fn move_paddle(
     time: Res<Time>,
 ) {
     let mut paddle_transform = query.single_mut();
-    let mut direction = 0.0;
 
-    if keyboard_input.pressed(KeyCode::ArrowLeft) {
-        direction -= 1.0;
-    }
-
-    if keyboard_input.pressed(KeyCode::ArrowRight) {
-        direction += 1.0;
-    }
+    let direction = if keyboard_input.pressed(KeyCode::ArrowLeft) {
+        -1.0
+    } else if keyboard_input.pressed(KeyCode::ArrowRight) {
+        1.0
+    } else {
+        0.0
+    };
```

Comment:
The behavior looks changed when user pressed `ArrowLeft` and `ArrowRight` simultaneously.🤔

Reply:
Yes, but I think pressing both buttons shouldn't make the player paddle stand still, it's not an important thing so feel free to merge or not merge it 😊

Reply:
They should definitely stand still: this is a common source of bugs when writing input handling code :)